FABulous/fabric_generator/file_parser.py

```diff
@@ -555,7 +551,8 @@ def parseTiles(fileName: pathlib.Path) -> tuple[list[Tile], list[tuple[str, str]
             else:
                 logger.error(f"Unknown tile description {temp[0]} in tile {t}.")
                 raise ValueError
-
+            # TODO: if old fabric.csv tileDir is set wrong
```

Comment:
Is the TODO supposed to be still in there to be fixed in the future? Or was it just missed?